packages/trader/src/Modules/Contract/Containers/contract-replay.tsx

```diff
@@ -0,0 +1,384 @@
+                                        is_accumulator_contract={is_accumulator}
+                                    />
+                                )}
+                            </MobileWrapper>
+                        </div>
+                    </React.Suspense>
+                </Div100vhContainer>
+            </PageOverlay>
+        </FadeWrapper>
+    );
+});
+
+export default ContractReplay;
+
+// CHART -----------------------------------------
+
+const ReplayChart = observer(
+    ({
+        is_dark_theme_prop,
+        is_accumulator_contract,
+    }: {
+        is_dark_theme_prop?: boolean;
+        is_accumulator_contract?: boolean;
+    }) => {
+        const trade = useTraderStore();
+        const { contract_replay, client, common, ui } = useStore();
+        const { contract_store, chart_state, chartStateChange, margin } = contract_replay;
+        const {
+            accumulator_previous_spot_time,
+            contract_config,
+            marker: accumulators_barriers_marker,
+            is_digit_contract,
+            barriers_array,
+            getContractsArray,
+            markers_array,
+            contract_info,
+        } = contract_store;
+        const { underlying: symbol, audit_details } = contract_info;
+        const allow_scroll_to_epoch = chart_state === 'READY' || chart_state === 'SCROLL_TO_LEFT';
+        const { app_routing_history, current_language, is_socket_opened } = common;
+        const { is_chart_layout_default, is_chart_countdown_visible } = ui;
+        const { end_epoch, chart_type, start_epoch, granularity } = contract_config || {};
+        const is_dark_theme = is_dark_theme_prop || ui.is_dark_mode_on;
+
+        /**
+         * TODO: remove forcing light theme once DBot supports dark theme
+         * DBot does not support for dark theme since till now,
+         * as a result, if any user come to report detail pages
+         * from DBot, we should force it to have light theme
+         */
+        const from_platform = getPlatformRedirect(app_routing_history);
+        const should_force_light_theme = from_platform.name === 'DBot';
+        const settings = {
+            language: current_language.toLowerCase(),
+            theme: is_dark_theme && !should_force_light_theme ? 'dark' : 'light',
+            position: is_chart_layout_default ? 'bottom' : 'left',
+            countdown: is_chart_countdown_visible,
+            assetInformation: false, // ui.is_chart_asset_info_visible,
+            isHighestLowestMarkerEnabled: false, // TODO: Pending UI
+        };
+        const scroll_to_epoch = allow_scroll_to_epoch && contract_config ? contract_config.scroll_to_epoch : undefined;
+        const all_ticks = audit_details ? audit_details.all_ticks : [];
+        const { wsForget, wsSubscribe, wsSendRequest, wsForgetStream } = trade;
+        const { is_beta_chart } = client;
+
+        const accu_barriers_marker_component = is_beta_chart
```

Comment:
Change logic a bit, previously it was:
`!is_beta_chart ? allMarkers[accumulators_barriers_marker?.type as keyof typeof allMarkers] : undefined`